.github/workflows/js-css-lint-test.yml

```diff
@@ -18,6 +18,7 @@ on:
       - 'package-lock.json'
       - '!feature-flags.json'
       - '!composer.json'
+      - '!.eslintrc.json'
```

Comment:
Looks like this was copied from the JS Tests YML, but we shouldn't ignore ESLint changes in this file anymore since this file is now responsible for running the ESLint task. 🙂 
```suggestion
```